iac/terraform/aws/web-client/cloudfront.tf

```diff
@@ -0,0 +1,79 @@
+resource "aws_cloudfront_distribution" "cdn" {
+  #checkov:skip=CKV_AWS_68: "CloudFront Distribution should have WAF enabled"
+  #checkov:skip=CKV2_AWS_47: "CloudFront Distribution should have WAF enabled"
+  #checkov:skip=CKV_AWS_86: "Ensure Cloudfront distribution has Access Logging enabled"
+  #checkov:skip=CKV2_AWS_32: "Ensure CloudFront distribution has a response headers policy attached"
+  origin {
+    domain_name = aws_s3_bucket.static_site.bucket_regional_domain_name
+    origin_id   = "S3PrimaryOrigin"
+
+    s3_origin_config {
+      origin_access_identity = aws_cloudfront_origin_access_identity.oai.cloudfront_access_identity_path
+    }
+  }
+
+  origin {
+    domain_name = aws_s3_bucket.cloudfront_failover_bucket.bucket_regional_domain_name
+    origin_id   = "S3FailoverOrigin"
+
+    s3_origin_config {
+      origin_access_identity = aws_cloudfront_origin_access_identity.oai.cloudfront_access_identity_path
+    }
+  }
+
+
+  enabled             = true
+  is_ipv6_enabled     = true
+  comment             = "CloudFront distribution for front-end site"
+  default_root_object = "index.html"
+
+  origin_group {
+    origin_id = "${var.environment}-OriginGroupId"
+
+    failover_criteria {
+      status_codes = [403, 404, 500, 502, 503, 504]
```

Comment:
missing 401, is that expected

Reply:
`401` included

Reply:
Couldn't include `401`. It's invalid status code for cloudfront.
![image](https://github.com/user-attachments/assets/b18f9a00-3f54-4be7-a149-f0cb0aa0db14)